Source/Tools/LinuxEmulation/LinuxSyscalls/Syscalls.h

```diff
@@ -255,7 +256,20 @@ class SyscallHandler : public FEXCore::HLE::SyscallHandler, FEXCore::HLE::Source
 protected:
   SyscallHandler(FEXCore::Context::Context* _CTX, FEX::HLE::SignalDelegator* _SignalDelegation);
 
-  fextl::vector<SyscallFunctionDefinition> Definitions {};
+  static auto constexpr cvt = [](auto in) {
+    union {
+      decltype(in) val;
+      void* raw;
+    } raw;
+    raw.val = in;
+    return raw.raw;
+  };
+
+  fextl::vector<SyscallFunctionDefinition> Definitions {std::max<std::size_t>(FEX::HLE::x64::SYSCALL_x64_MAX, FEX::HLE::x32::SYSCALL_x86_MAX),
+                                                        {
+                                                          .NumArgs = 255,
+                                                          .Ptr = cvt(&UnimplementedSyscall),
+                                                        }};
```

Comment:
Can we move this to the constructor instead? I'd prefer the funky lambda to remain local state in a function instead of being a visible class member. I don't think `std::vector` allocates any memory on default construction, so I don't think there's a difference in allocation behavior either way.

Reply:
Moved to constructor.